server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -1070,21 +1071,30 @@ private void cleanUpCompactors() {
       var groups = zoorw.getChildren(compactorQueuesPath);
 
```

Comment:
If I checked MANAGER_COMPACTION_SERVICE_PRIORITY_QUEUE_SIZE_FACTOR property here, then I could remove it from the fixed property set.

Reply:
Moved where this was being done and removed from fix property set in 18d2243